buildSrc/src/main/groovy/org/elasticsearch/gradle/precommit/PrecommitTasks.groovy

```diff
@@ -109,4 +110,22 @@ class PrecommitTasks {
         }
         return checkstyleTask
     }
+
+
+    private static Task configureLoggerUsage(Project project) {
+        Task loggerUsageTask = project.tasks.create('loggerUsageCheck', LoggerUsageTask.class)
+
+        project.configurations {
```

Comment:
We've been moving towards a more java like style in buildSrc. Can you use the direct methods to create configurations/dependencies, instead of closures?